### ISSUE TITLE
FR: A group with a single item becomes a button with no dropdown

### ISSUE THREAD

**Issue by BenParizek** [angell-co/Spoon#16]
I expect you considered this and there may be challenges to doing so, but if it becomes possible at some point, it'd be nice to see single items in groups be active buttons without the additional step of a dropdown with one item in it.

A rough mockup just for conversation:
http://share.barrelstrength.co/1n0m320u1D3V

Perhaps this update relates to this issue:
https://github.com/angell-co/Spoon/issues/12

**Comment by kisonay** [comment]
Agreed. If A group only has 1 item in it have it a single item and not a group. That would give you the database refenence to store the settings.
